src/test/validation_tests.cpp

```diff
@@ -20,6 +24,21 @@ void SortTxs(CBlock &block, bool reverse = false) {
     std::reverse(block.vtx.begin() + 1, block.vtx.end());
   }
 }
+struct Fixture {
+  std::unique_ptr<blockchain::Behavior> blockchain_behavior = blockchain::Behavior::NewForNetwork(blockchain::Network::test);
```

Comment:
Nit: This is a very long line, so that GitHub is actually not showing it all at once. Would be better to add a line break.

Reply:
Fixed in https://github.com/dtr-org/unit-e/pull/1080/commits/439f587e8a952a906d5aa3471feebb72b67f6780